WKData/Sources/WKData/Models/Donate/WKDonateConfig.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+public struct WKDonateConfig: Codable {
+    let version: Int
+    public let currencyMinimums: [String: Decimal]
+    public let currencyMaximums: [String: Decimal]
+    public let currencyAmounts7: [String: [Decimal]]
```

Comment:
I understand these currently match the API, but just wanted to mention the same note as in Slack about potentially more precise naming here if those suggestions still agree with you.

Reply:
Done! I updated the mock json and the config at donate.wikimedia.org...so far I haven't changed the config at test.wikipedia.org. I will EOD today once I know FR Tech isn't testing against it.